CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ The following notes provide some details on what has been revised for each
 version in reverse chronological order (most recent version at the top
 of the list).
 
+3.1.0 (unreleased)
```

Comment:
I am not sure what is going on here (currently github says it is a merge conflict), but both `3.1.0` and `3.0.2` are "unreleased". I think `3.0.2` should have been updated with the release date when it was being planned to be released but if it wasn't, then maybe it should be done now?

Reply:
The code was 'released' starting 15-July (or thereabouts), but the change log did not get updated to reflect that...until now.